lua/yaml-companion/lsp/util.lua

```diff
@@ -19,6 +19,12 @@ M.get_all_yaml_schemas = function()
     require("yaml-companion.config").options.schemas.result or {}
   )
 
+  for i, schema in ipairs(schemas) do
```

Comment:
Not sure where the weird `result` comes from, but we definitely want to show all URIs, not only those starting with `https`.

Reply:
take a closer look at the regular expression. the `s?` is conditional.
it could be either `http` or `https`, and honestly, a URL without a `http/s` is not a URL, also there's no case in the returned schemas where there is a valid URL without its prefix, it's only for the weird "result" result.

Reply:
The problem is that the URI could also be a local file or a remote ftp for example. Do you know where "result" result may be coming from? Is it an issue in yaml-companion or the language server? 

I cannot reproduce it on my end with nvim master and yaml-language-server 1.10.0.

Reply:
Removed it.